### PR TITLE
revise lesson to visualize accessing and updating object state

### PR DESCRIPTION
**Pull request by linda-seiter**

### CODE REVIEW

**Review by developerjaym — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,32 +1,288 @@
...
+
+1. Set a breakpoint on the first line of executable code in the `main` method (i.e. not a comment or blank line),
+   then click the "debug" icon to launch the debugger.   Your line number may differ depending on comments and blank lines.  
+   
+   ![dog breakpoint](https://curriculum-content.s3.amazonaws.com/6676/java-mod2-oop-fundamentals/set_breakpoint.png)
+
+2. Select the "Java Visualizer" tab in the debug window.
+   Notice the call stack shows the `main` method has stopped at the breakpoint (your line number may be different).    
+ 
+   ![java visualizer tab](https://curriculum-content.s3.amazonaws.com/6676/java-mod2-oop-fundamentals/breakpoint_reached.png)
```
Comment (kcleland0818) [inline README.md:154]: Maybe adjust the screenshot with the folders in the project structure collapsed? That way students don't think they need to also have a `Cat` and a `Bike` class?

Comment (kcleland0818) [reply README.md:154]: The project name might also throw students?

Comment (linda-seiter) [reply README.md:154]: yes, thanks.

Comment (linda-seiter) [reply README.md:154]: I updated the image to show a sample project like the student's.

**Review by developerjaym — APPROVED** [review]

**Review by AlveeM — APPROVED** [review]

**Pull request merged**